src/components/SingleName/ResolverAndRecords/Records.js

```diff
@@ -143,24 +143,23 @@ const useGetRecords = domain => {
   const { loading: addressesLoading, data: dataAddresses } = useQuery(
     GET_ADDRESSES,
     {
-      variables: { name: domain.name, keys: coinList },
-      skip: !coinList,
+      variables: {
+        name: domain.name,
+        keys: _.union(coinList, COIN_PLACEHOLDER_RECORDS)
+      },
       fetchPolicy: 'network-only'
     }
   )
-
   const { loading: textRecordsLoading, data: dataTextRecords } = useQuery(
     GET_TEXT_RECORDS,
     {
       variables: {
         name: domain.name,
-        keys: resolver && resolver.texts
+        keys: _.union(resolver && resolver.texts, TEXT_PLACEHOLDER_RECORDS)
```

Comment:
Will this work if `resolver` is not set? Won't it try and take the union of `undefined` and a list?

Reply:
no, `.union()` merge an array and it also get rid of null or undefined (which I wasn't sure so it was good to double check).

```
_.union([0,1,2], [1,2,3])
(4) [0, 1, 2, 3]
_.union(null, [1,2,3])
(3) [1, 2, 3]
_.union(undefined, [1,2,3])
(3) [1, 2, 3]
```